coalib/misc/Shell.py

```diff
@@ -1,14 +1,7 @@
-from contextlib import contextmanager
-import functools
 import shlex
-from subprocess import PIPE, Popen, call, DEVNULL
-
-
-call_without_output = functools.partial(call, stdout=DEVNULL, stderr=DEVNULL)
-"""
-Uses subprocess.call to execute a command, but suppresses the output and
-the errors.
-"""
+from contextlib import contextmanager
```

Comment:
hmm this reorder should not have been needed. Can you try reverting it(i.e move the context manager one up)?